src/test/scala/org/zalando/kanadi/SubscriptionsSpec.scala

```diff
@@ -35,49 +28,31 @@ class SubscriptionsSpec(implicit ec: ExecutionEnv) extends Specification with Co
     EventTypes(nakadiUri, None)
   val subscriptionIds: mutable.ParSet[SubscriptionId] = mutable.ParSet.empty
 
-  eventTypeName.pp
-  s"Consumer Group: $consumerGroup".pp
-
-  def createEventType = eventsTypesClient.create(EventType(eventTypeName, OwningApplication, Category.Business))
```

Comment:
This was causing issues when run in `beforeAll` (likely due to scalatest not expecting to do async calls like this before the tests are run) so I just moved the logic into the main test and I also cleaned up the test so its easier to read

Reply:
aha, got it